addon/utils.js

```diff
@@ -404,3 +405,32 @@ export function isModelPathValid (path, bunsenModel) {
 
   return Boolean(bunsenModel)
 }
+
+/**
+ * Aggregates paths to internal model values.
+ *
+ * @export
+ * @param {any} val Value to check for internal values
+ * @returns {String[]} List of strings that are paths to internal values
+ */
+export function findInternalValues (val) {
+  return _.chain(val)
+    .map(function (item, key) {
+      if (key === '_internal') {
+        return key
+      }
+      if (['boolean', 'number', 'string', 'undefined', 'null'].indexOf(typeof item) !== -1) {
```

Comment:
I've been using `[].includes` to check existence in an array more than `indexOf`.